code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -1058,6 +1058,25 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	if(!target)
 		return 0 //wait where'd they go
 	clockwork_generals_invoked["sevtug"] = world.time + CLOCKWORK_GENERAL_COOLDOWN
+
+	// You can throw off the mind control if you're shielded
+	for(var/obj/item/weapon/implant/mindshield/M in target)
+		if(M.implanted)
+			M << "<span class='sevtug'>Bu, ybbx, n zvaqfuvryq. Phgr, ohg \
```

Comment:
Incidentally this is hilarious if you translate it sevtug is literally bullshitting the invoker